tests/integration/test_s3_cache_locality/test.py

```diff
@@ -70,26 +71,31 @@ def started_cluster():
 
         yield cluster
     finally:
-        shutil.rmtree(os.path.join(SCRIPT_DIR, "data/generated/"))
+        shutil.rmtree(os.path.join(SCRIPT_DIR, "data/generated/"), ignore_errors=True)
         cluster.shutdown()
 
 
-def check_s3_gets(cluster, node, expected_result, cluster_first, cluster_second, enable_filesystem_cache):
+def check_s3_gets(cluster, node, expected_result, cluster_first, cluster_second, enable_filesystem_cache, lock=False):
```

Comment:
Here and everywhere else down from here, just for the sake of readability and maintainability... please you consider renaming `lock` into something like `use_lock_object_storage_task_distribution_ms` or even just (`lock_object_storage_task_distribution_ms` with either `None` or numerical value )?

So it is clear what that parameter is used for, and also more `grep` friendly...